pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "databricks-ai-bridge"
-version = "0.8.0.dev0"
+version = "0.8.1"
```

Comment:
Can we do this when we release and cut a branch?

Reply:
Oh yes sorry not ready for review yet, just needed this to install it on serving and test